webapps/asterics-camerainput-eyecontrol/index.html

```diff
@@ -0,0 +1,287 @@
+<!DOCTYPE html>
```

Comment:
shouldn't this file be somehow inside the included submodule and then maybe be copied by ant to this directory? If I understand it correctly, this file now is only existing in this branch gh-pages. Isn't it also useful in the submodule `asterics-camerainput-eyecontrol` for being used as standalone solution?

Reply:
the same question applies to the file `webapps/asterics-camerainput-eyecontrol/models/EyeControlledMouse(W).acs`.

Reply:
that is actually the case.
the build.xml file of the root directory copies the relevant files of the webapp (only the webapp part) to the webapps/ folder. But there you have to commit them because otherwise the online webpage (http://asterics.github.io/AsTeRICS/webapps/startpage/) won't be updated.

Reply:
ah, yes, makes sense! thanks for the explanation 👍